### PR TITLE
Refactor irc2as to made code climate a little more happy

### PR DESCRIPTION
**Pull request by silverbucket**
General code separation, cleanup, and modularization. It's not perfect, but it's an improvement.

### CODE REVIEW

**Comment by silverbucket** [comment]
If anyone is around to give a quick review so I can merge that'd be cool. this module has unit tests so I'm pretty confident nothing broke. cc @galfert @gregkare @raucao

File: packages/irc2as/index.js [hunk 1]
```diff
@@ -484,14 +249,15 @@ IrcToActivityStreams.prototype.input = function (string) {
...
+        ase.presence(nick, role, channel);
         break;
 
         /** */
-        default:
-        this.events.emit(EVENT_UNPROCESSED, string);
-        break;
+        // default:
+        // this.events.emit(EVENT_UNPROCESSED, string);
+        // break;
```
Comment (galfert) [inline packages/irc2as/index.js]: Why is this commented out? Weren't we talking about this in the last Kosmos call, saying it's good to have?

File: packages/irc2as/test/test-suite.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ if (typeof(define) !== 'function') {
 
 function matchStream(env, test) {
     return (stream) => {
+        // console.log(stream);
```
Comment (galfert) [inline packages/irc2as/test/test-suite.js]:
```suggestion
```

**Review by galfert — APPROVED** [review]
Nice refactor. Code looks good to me.

Just left a small question about some commented out part.

**Comment by silverbucket** [comment]
@galfert both of your points were addressed, thanks!

**Pull request merged**